### PR TITLE
[YOLO examples] moving v3 examples to unified YOLO examples directory

### PR DESCRIPTION
**Pull request by bfineran**
* support a single location for YOLOv3 and YOLOv5 examples since most code can be shared between the two
* support for generic anchor grids will be added in a follow up PR
* YOLOv5 model stubs and examples to be added after v5 lands in SparseZoo

### CODE REVIEW

File: examples/ultralytics-yolo/README.md [hunk 1]
```diff
@@ -14,24 +14,43 @@ See the License for the specific language governing permissions and
...
 ```
 
+## SparseZoo Stubs
+The following examples may be run with local ONNX YOLO models, or by using pre-trained, pre-sparsified YOLO models
+from the [SparseZoo](https://sparsezoo.neuralmagic.com/).
+
+The [SparseZoo](https://sparsezoo.neuralmagic.com/) contains both 
+baseline and models sparsified to different degrees of inference performance vs. baseline loss recovery.
+
+Each model in the SparseZoo has a specific stub that identifies it, any YOLO model stub in the SparseZoo can be used to
```
Comment (jeanniefinks) [inline examples/ultralytics-yolo/README.md:41]:
```suggestion
Each model in the SparseZoo has a specific stub that identifies it; any YOLO model stub in the SparseZoo can be used to
```

File: examples/ultralytics-yolo/deepsparse_utils.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 """
-Utilities for Yolo V3 pre and post processing for DeepSparse pipelines
+Utilities for YOLO pre and post processing for DeepSparse pipelines
```
Comment (jeanniefinks) [inline examples/ultralytics-yolo/deepsparse_utils.py:16]:
```suggestion
Utilities for YOLO pre- and post-processing for DeepSparse pipelines
```

File: examples/ultralytics-yolo/deepsparse_utils.py [hunk 3]
```diff
@@ -356,7 +356,7 @@ def load_image(
 
 class YoloPostprocessor:
     """
-    Class for performing postprocessing of YOLOv3 model predictions
+    Class for performing postprocessing of YOLO model predictions
```
Comment (jeanniefinks) [inline examples/ultralytics-yolo/deepsparse_utils.py:359]:
```suggestion
    Class for performing post-processing of YOLO model predictions
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
There were some other edits I noticed in README; decide if you want to keep them in scope here or file a separate PR, make it so.

- Search for the phrase: "or glog pat" - have no idea what that is?
- Update this line to be: Note for quantized performance, your CPU must support VNNI instructions. Review `/proc/cpuinfo` for the flag `avx512_vnni` to verify chipset compatibility.
- Under "Server" - we had enhanced the content per the "Server" section of this file:
https://github.com/neuralmagic/deepsparse/blob/main/examples/classification/README.md

**Comment by bfineran** [comment]
@jeanniefinks added your suggestions into this PR

**Review by finkatronic — APPROVED** [review]
lgtm!

**Comment by jeanniefinks** [comment]
> lgtm!

Ugh, was logged with other identity. Will try to fix this

**Review by jeanniefinks — APPROVED** [review]

**Review by mgoin — APPROVED** [review]

**Review by markurtz — APPROVED** [review]
Note that we'll need to keep an eye on the links for updating so we don't end up with dead ones for the model page and readmes

**Comment by bfineran** [comment]
@markurtz noted, will go through the sparseml side of things

**Pull request merged**